src/rsvc/README.md

```diff
@@ -16,15 +16,15 @@ Raft adopts a strong leadership design, so does each replicated service. A servi
 
 A replicated service is implemented using a stack of modules:
 
-	[ mgmt_svc, pool_svc, ... ]
+	[ mgmt_svc, pool_svc, cont_svc, ... ]
```

Comment:
I omitted cont_svc deliberately to make the "]"s align vertically, thus the "...". :) But it's okay to add it back in.